dashboards/pmm-list.json

```diff
@@ -181,7 +181,7 @@
         ]
     },
     "timezone": "browser",
-    "title": "_PMM List",
+    "title": "_PMM RDS and Remote Instances",
```

Comment:
IMHO we should rename the file to something more related to the new title.

e.g.
_PMM Add Instance -> pmm-add-instance.json
_PMM Query Analytics -> pmm-qan.json
_PMM Query Analytics Settings -> pmm-settings.json
_PMM System Summary -> pmm-summary.json

Reply:
So there will be five elements on the PMM > Query Analytics menu now:
1. Add instance
2. RDS and Remote instances (list instances)
3. Query ANalytics
4. QAN settings
5. System Summary

Reply:
no, let's not  change  name  of the file for now